blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""Models for Blog app."""
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Create Post model for Blog app."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish date method."""
```

Comment:
What does it do?

Reply:
Saves the publication date?

Reply:
No, it publishes the post.

Reply:
I should write """Publish the post.""" ?

Reply:
yes

Reply:
or "mark post as published"